yolapy/resources/cname_zone.py

```diff
@@ -0,0 +1,10 @@
+class CnameZoneMixin(object):
+    """Methods for managing Cname Zones."""
+    def create_cname_zone(self, hostname, partner_id):
+        return self.post(
+            self._cname_zones_path(),
+            json={'partner_id': partner_id, 'hostname': hostname}).json()
+
+    def _cname_zones_path(self, *parts):
+        path = '/'.join(['cname-dns-zones'] + list(parts))
+        return '/%s/' % path
```

Comment:
I prefer to have a path as a string...  It's much clearer what it is.  And I see no benefits of generating it in this way.